entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+echo "========== Downloading single HTML =================="
+php /app/cli.php download:single-html
+echo "============ Converting to e-book ===================="
+ebook-convert shape-up.html /app/output/shape-up.${1} \
```

Comment:
While I see the intention behind this, I would rather not bundle this functionality with the library: When it comes to books, rights are a bit complicated. I can see a future where 37signals decides to sell an epub version of the file and I don't want it to seem like I am trying to do the same thing for free.

The scope of this library should only be "if you want this offline, here you go".

While converting it automatically probably is fine, I just don't want that risk in this repository. I hope you understand.